config/initializers/ruby_stackoverflow.rb

```diff
@@ -0,0 +1,5 @@
+RubyStackoverflow.configure do|config|
+  config.client_key = ''
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.